### PR TITLE
Sort language in dropdown alphabetically

### PR DESCRIPTION
**Pull request by haritsE**
# Summary
Sort languages in the dropdown alphabetically.
Issue: https://github.com/quran/quran.com-frontend/issues/57

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32820

**Comment by haritsE** [comment]
Just realized that the `.sort((current, next) => current.name - next.name);` in the `slugs` declaration doesn't work. Should I put the function there instead?

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32821

File: src/components/ContentDropdown/index.js [hunk 1]
```diff
@@ -4,6 +4,22 @@ import MenuItem from 'react-bootstrap/lib/MenuItem';
 
 const style = require('./style.scss');
 
+const compareAlphabetically = property =>
+  (a, b) => {
```
Comment (mmahalwy) [inline src/components/ContentDropdown/index.js]: perhaps make it more specific, as in `(previous, next)` ?

**Comment by mmahalwy** [comment]
@haritsE that would be awesome!

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32822

**Comment by mmahalwy** [comment]
@haritsE let me know when this is ready :)

**Comment by haritsE** [comment]
@mmahalwy done 👍

**Comment by mmahalwy** [comment]
Thank you!

**Pull request merged**